packages/editor/src/components/post-title/index.native.js

```diff
@@ -42,6 +48,14 @@ class PostTitle extends Component {
 		this.props.onUpdate( title );
 	}
 
+	onEnter( ) {
```

Comment:
We could remove this method entirely and instead copy (and call) [this method](https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/post-title/index.js#L177) from the web .js sources.

I've tried it locally and it works and it works.

The advantage of using the same code as the web .js file is that it should make it easier for us to maintain this sourcefile if there are changes, or eventually unify it completely.